src/lib/PartitionedGraphs/src/abstractpartitionedgraph.jl

```diff
@@ -11,54 +13,136 @@ using Graphs:
     vertices
 using ..NamedGraphs: NamedGraphs, AbstractNamedGraph
 using ..NamedGraphs.GraphsExtensions:
-    GraphsExtensions, add_vertices!, not_implemented, rem_vertices!
+    GraphsExtensions, add_vertices!, not_implemented, rem_vertices!, subgraph, vertextype
 
-abstract type AbstractPartitionedGraph{V, PV} <: AbstractNamedGraph{V} end
+# For you own graph type `g`, you should define a method for this function is you
+# desire custom partitioning.
+partitioned_vertices(g::AbstractGraph) = [vertices(g)]
 
-#Needed for interface
-partitions_graph(pg::AbstractPartitionedGraph) = not_implemented()
-unpartitioned_graph(pg::AbstractPartitionedGraph) = not_implemented()
-function unpartitioned_graph_type(pg::Type{<:AbstractPartitionedGraph})
-    return not_implemented()
+# For fast quotient edge checking and graph construction, one should overload this function.
+function quotient_graph(g::AbstractGraph, pvs = nothing)
+    if isnothing(pvs) === nothing
+        pvs = partitioned_vertices(g)
+    end
```

Comment:
```suggestion
function quotient_graph(g::AbstractGraph, pvs = partitioned_vertices(g))
```
Just making it shorter (and I think a bit clearer).

Reply:
See https://github.com/ITensor/NamedGraphs.jl/pull/110/commits/43c89da662ce0f53a801309c2b397bcc178eca38